docs/administrator.md

```diff
@@ -685,6 +685,31 @@ aws_or_gcp:
 ...
 ```
 
+### Setup pod environment configmap
```

Comment:
I believe the documentation part needs to be formulated differently.

* Title of the section should include "GCS" to indicate it's not just general configmap.

*  The operator doesn't have any preferences or suggestions about which method to use, so "we want to use WAL-G" need to be omitted. Instead it could be formulated as "To make operator work with GCS do the following: ...".

Reply:
this paragraph is part of a bigger section about GCP so title is fine.

Reply:
Indeed, then the title part is fine. But I would still ask to avoid indirect suggestions about using WAL-G (it could be a note though). Other than that looks good.

Reply:
@erthalion I re-worded that section based on your feedback. I left a note as to why we are using WAL-G instead of WAL-E, so that end-user understands the reasoning, instead of just using "magical" configmap. Let me know if you agree with the changes.

Reply:
@sagor999 Yes, looks good, thanks. Do you have any references we can also add to make the claim look more respectable?

Reply:
@erthalion valid point. I looked it up, and it seems like it is possible to make WAL-E work with GCS. So I have re-worded that phrase.